apps/web/playwright/fixtures/users.ts

```diff
@@ -107,17 +107,20 @@ const createTeamAndAddUser = async (
     isOrgVerified,
     hasSubteam,
     organizationId,
+    index,
   }: {
     user: { id: number; email: string; username: string | null; role?: MembershipRole };
     isUnpublished?: boolean;
     isOrg?: boolean;
     isOrgVerified?: boolean;
     hasSubteam?: true;
     organizationId?: number | null;
+    index?: number;
   },
   workerInfo: WorkerInfo
 ) => {
-  const slug = `${isOrg ? "org" : "team"}-${workerInfo.workerIndex}-${Date.now()}`;
+  const slugIndex = index ? `-count-${index}` : "";
```

Comment:
If an index is passed in (when called in loop we append it to the slug to prevent duplicates for this run)